lib/testscript_engine/validation/validation.rb

```diff
@@ -33,7 +33,9 @@ def change_validator_endpoint(endpoint)
   # endpoint, and return whether the response conveys validation errors
   def valid_resource?(resource, *profiles)
     initial_logger = FHIR.logger
-    FHIR.logger = Logger.new('/dev/null')
+    #FHIR.logger = Logger.new('/dev/null')
+    #FHIR.logger = Logger.new(RUBY_PLATFORM != 'i386-mingw32' ? '/dev/null' : 'NUL')
+    FHIR.logger = Logger.new(STDOUT)
```

Comment:
make this a config option, e.g., --verbose?

Reply:
The previous one './dev/null' cause error when I tested in Window machine. I understand we don't have QA but STDOUT worked in general more. It's good idea to add this option! I will do it.

Reply:
Done : https://github.com/fhir-crucible/testscript-engine/pull/70/commits/47d222a37b77e7555e9f9b777a1183fcc586d80c